pom.xml

```diff
@@ -514,5 +520,6 @@
         <log4j.core.version>2.8.2</log4j.core.version>
         <!--Dashboard dependencies-->
         <org.codehaus.mojo.version>1.3.2</org.codehaus.mojo.version>
+        <findbugs.annotations.version>3.0.1</findbugs.annotations.version>
```

Comment:
Lets add a comment above `<!--  Findbugs dependencies -->

Reply:
This is Fixed!